webpack.config.js

```diff
@@ -33,22 +33,26 @@ module.exports = ( env ) => {
         ),
         'test-helpers': './test-helpers.ts'
       },
-      (env.element ? {
-        'main': `./packages/${ env.element }/index.ts`,
+      (env.element ? ifProd({
```

Comment:
I really don't want to be picky but we should create some helper function for this because it's really messy. If you want we can pair to come with some viable solution.